metadata-integration/java/datahub-client/src/main/java/datahub/client/RestEmitter.java

```diff
@@ -68,18 +82,25 @@ public static boolean makeRequest(String urlStr, String method, String payloadJs
 
   }
 
-  public boolean testConnection() {
-    try {
-      RESTEmitter.makeRequest(this.gmsUrl + "/config", "GET", null);
+  public boolean testConnection() throws IOException {
+      this.makeRequest(this.gmsUrl + "/config", "GET", null);
       return true;
+  }
 
-    } catch (IOException e) {
-      e.printStackTrace();
-      return false;
-    }
+  public static RestEmitter create(String gmsUrl) {
+    return new RestEmitter(gmsUrl, DEFAULT_CONNECT_TIMEOUT_SEC, DEFAULT_READ_TIMEOUT_SEC, null);
+  }
+
+  public static RestEmitter create(String gmsUrl, int connectTimeoutSec, int readTimeoutSec) {
```

Comment:
I'm assuming we have unit tests for this class (and McpEmitter?)

Reply:
We have unit tests at spark-lineage level which covers both of these classes.